.gitignore

```diff
@@ -0,0 +1,4 @@
+1.4/Assemblies
+1.4/Source/WWE/.vs
```

Comment:
wrong name, dont use WWE, use ATE instead

Reply:
Fixed in the new gitignore